sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -129,6 +129,7 @@ export interface EventHubBufferedProducerClientOptions extends EventHubClientOpt
 // @public
 export interface EventHubClientOptions {
     customEndpointAddress?: string;
+    identifier?: string;
```

Comment:
We'll want to have a way for each type to expose this so that callers can inspect it.   For example, in .NET, we have an [Identifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs#LL95C10-L95C10) property on each type.